test/tests_for_db/ee2_MongoUtil_test.py

```diff
@@ -57,6 +58,39 @@ def test_init_ok(self):
         mongo_util = self.getMongoUtil()
         self.assertTrue(set(class_attri) <= set(mongo_util.__dict__.keys()))
 
+    def test_update_jobs_success(self):
```

Comment:
I don't quite undestand this test - it just saves a couple jobs and that's it?

Reply:
Looks like either I forgot to write this test or deleleted it?? Hmm. Updated it

Reply:
:+1: